BGK.Data/Repositories/AdminRepository.cs

```diff
@@ -52,5 +52,14 @@ public bool ChangePasswordAdmin(Admin newItem)
             _db.SaveChanges();
             return true;
         }
+
+        public bool RecoveryAdminById(int id)
+        {
+            var res = _db.Admin.FirstOrDefault(x => x.Id == id);
+
+            res.IsDeleted = true;
+            _db.SaveChanges();
+            return true;
+        }
```

Comment:
1. please rename this method properly (RecoverAdminById instead of RecoveryAdminById)
2. use GetInstance method instead of "new ApplicationContext()" at the top of this class

Reply:
Done